web/src/views/Search/SearchSidebar.vue

```diff
@@ -183,6 +185,41 @@ export default defineComponent({
     }
     watch(filterText, updateSearch);
 
+    function trackFilterConditions(val: Condition[], oldVal: Condition[]) {
```

Comment:
What do you think about changing `val` and `oldVal` to be something like `newConditionList` and `oldConditionList`? I think that might make it easier to read through this function.